Software/Development/SemesterProject/App/encoderManager.c

```diff
@@ -177,7 +177,7 @@ void rotorSpeedCalc(void)
     deltaTheta = obtainDeltaTheta();
 
     //Speed is only calculated every 10 degrees.
-    if (deltaTheta > DEG_10_TO_RAD)
+    if (deltaTheta > DEG_10_TO_RAD || rotorPosSpeedObject.speedTempCount > 120 * SW_PERIOD_US)
```

Comment:
I have made changes in encoderManager and this now has a slightly different way to be called, thus i will change it in my part and you can take it out from here, sorry about that...

Reply:
I will take it out then